crates/routes/src/feeds.rs

```diff
@@ -273,12 +270,22 @@ async fn get_feed_front(
   .list(&site_view.site, &mut context.pool())
   .await?;
 
-  let protocol_and_hostname = context.settings().get_protocol_and_hostname();
+  let title = format!("{} - Subscribed", site_view.site.name);
+  let link = context.settings().get_protocol_and_hostname();
   let items = create_post_items(posts, context.settings())?;
+  Ok(send_feed_response(title, link, items, site_view))
+}
+
+fn send_feed_response(
+  title: String,
+  link: String,
+  items: Vec<Item>,
+  site_view: SiteView,
+) -> HttpResponse {
```

Comment:
This is missing the optional description, which you removed above for community. Could also be added for person and multi_community.

L294 Shouldn't always be setting the site description IMO. You could use the site description as the backup, ie

```rust
Channel {
  ...
  description: desc.unwrap_or(site_view.site.description)
```